app/labor/rate_limit_checker.rb

```diff
@@ -9,22 +9,13 @@ def initialize(user = nil)
     @user = user
   end
 
-  class UploadRateLimitReached < StandardError; end
-  class DailyFollowAccountLimitReached < StandardError; end
+  UploadRateLimitReached = Class.new(StandardError)
```

Comment:
Semantically these two forms are the same (`class` is essentially syntactic sugar for creating class objects and assigning them to constants), but the previous one always felt a bit noisy to me for 1 line class definitions.